crates/wit-dylib/ffi/src/ffi.rs

```diff
@@ -159,6 +160,15 @@ pub struct wit_list {
 pub type wit_list_t = wit_list;
 #[repr(C)]
 #[derive(Debug, Copy, Clone)]
+pub struct wit_map {
+    pub interface: *const ::std::os::raw::c_char,
+    pub name: *const ::std::os::raw::c_char,
+    pub key_ty: wit_type_t,
+    pub value_ty: wit_type_t,
+}
+pub type wit_map_t = wit_map;
+#[repr(C)]
+#[derive(Debug, Copy, Clone)]
```

Comment:
For this I'd recommend skipping wit-dylib entirely. I ended up skipping fixed-size-lists mostly for example and figure it's fine to come back later and add support as needed.